src/eva/core/data/datasets/embeddings.py

```diff
@@ -65,6 +66,8 @@ def __init__(
 
         self._data: pd.DataFrame
 
+        multiprocessing.set_start_method("spawn", force=True)
+
```

Comment:
maybe we can turn it into a method `_set_multiprocessing_method`, where in the function doc you write a bit what it going on? like

```
def __init__(self, ...)
  self._set_multiprocessing_method()
  
def _set_multiprocessing_method(self):
  """
  On CUDA devices we need to use the `spawn` method etc...
  """
```

Reply:
Good point. Now put this within a private method `_set_multiprocessing_start_method`.